processing/src/main/java/org/apache/carbondata/lcm/status/SegmentStatusManager.java

```diff
@@ -246,15 +247,22 @@ private Integer compareDateValues(Long loadValue, Long userValue) {
    * updates deletion status
    * @param loadIds
    * @param tableFolderPath
+   * @param dbName
+   * @param tableName
    * @return
    */
-  public List<String> updateDeletionStatus(List<String> loadIds, String tableFolderPath) {
-    ICarbonLock carbonLock = CarbonLockFactory
+  public List<String> updateDeletionStatus(List<String> loadIds, String tableFolderPath,
+      String dbName, String tableName) throws Exception {
```

Comment:
I think no need to to change the method signature here...database name and table name can be taken from carbontable identifier and carbonTableIdentifier object is available in the mehod